fvtest/compilerunittest/p/BinaryEncoder.cpp

```diff
@@ -2331,6 +2331,12 @@ INSTANTIATE_TEST_CASE_P(VMX, PPCTrg1Src2EncodingTest, ::testing::ValuesIn(*TRTes
     std::make_tuple(TR::InstOpCode::vmuluwm,  TR::RealRegister::vr31, TR::RealRegister::vr0,  TR::RealRegister::vr0,  0x13e00089u),
     std::make_tuple(TR::InstOpCode::vmuluwm,  TR::RealRegister::vr0,  TR::RealRegister::vr31, TR::RealRegister::vr0,  0x101f0089u),
     std::make_tuple(TR::InstOpCode::vmuluwm,  TR::RealRegister::vr0,  TR::RealRegister::vr0,  TR::RealRegister::vr31, 0x1000f889u),
+    std::make_tuple(TR::InstOpCode::vmuleub,  TR::RealRegister::vr31, TR::RealRegister::vr0,  TR::RealRegister::vr0,  0x13e00208u),
+    std::make_tuple(TR::InstOpCode::vmuleub,  TR::RealRegister::vr0,  TR::RealRegister::vr31, TR::RealRegister::vr0,  0x101f0208u),
+    std::make_tuple(TR::InstOpCode::vmuleub,  TR::RealRegister::vr0,  TR::RealRegister::vr0,  TR::RealRegister::vr31, 0x1000f208u),
+    std::make_tuple(TR::InstOpCode::vmuloub,  TR::RealRegister::vr31, TR::RealRegister::vr0,  TR::RealRegister::vr0,  0x13e00008u),
+    std::make_tuple(TR::InstOpCode::vmuloub,  TR::RealRegister::vr0,  TR::RealRegister::vr31, TR::RealRegister::vr0,  0x101f0008u),
+    std::make_tuple(TR::InstOpCode::vmuloub,  TR::RealRegister::vr0,  TR::RealRegister::vr0,  TR::RealRegister::vr31, 0x1000f008u),
```

Comment:
gas encodes this instruction as 0x1000f808 (with f8 instead of f0). The value here decodes to `vmuloub vr0, vr0, vr30` according to objdump